aws/resource_aws_api_gateway_vpc_link.go

```diff
@@ -0,0 +1,200 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayVpcLink() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayVpcLinkCreate,
+		Read:   resourceAwsApiGatewayVpcLinkRead,
+		Update: resourceAwsApiGatewayVpcLinkUpdate,
+		Delete: resourceAwsApiGatewayVpcLinkDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"target_arn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
```

Comment:
Is there any particular reason this should be implemented as 1-1 relationship instead of reflecting the API, i.e. allowing users to assign multiple LBs to one VPC link?